openfisca_core/entities.py

```diff
@@ -281,15 +285,15 @@ def init_from_json(self, entities_json):
 
         Entity.init_from_json(self, entities_json)
```

Comment:
Sorry, but I'm not sure I understand your suggestion. Could you please rephrase it ?